op-chain-ops/cmd/celo-migrate/state.go

```diff
@@ -264,9 +267,11 @@ func applyAllocsToState(db *state.StateDB, genesis *core.Genesis, config *params
 						log.Info("Account already exists with different code and is whitelisted, overwriting...", "address", k)
 					} else {
 						log.Warn("Account already exists with different code and is not whitelisted, overwriting...", "address", k, "oldCode", db.GetCode(k), "newCode", v.Code)
+						return fmt.Errorf("account already exists with different code and is not whitelisted: %s", k.Hex())
```

Comment:
This branch seems redundant, seems to be the same as on line 274 (same account, different code, not whitelisted)

Reply:
I kept them separate to make the error of not supplying a whitelist more explicit.